cmd/aries-agentd/startcmd/start.go

```diff
@@ -21,61 +23,69 @@ import (
 
-	// AgentInboundHostFlagShorthand is the flag shorthand for the agent inbound host command line argument.
-	AgentInboundHostFlagShorthand = "i"
+	agentInboundHostFlagName = "inbound-host"
 
-	// AgentInboundHostFlagUsage is the usage text for the agent inbound host command line argument.
-	AgentInboundHostFlagUsage = "Inbound Host Name:Port. This is used internally to start the inbound server."
+	agentInboundHostFlagShorthand = "i"
+
+	agentInboundHostFlagUsage = "Inbound Host Name:Port. This is used internally to start the inbound server." +
+		" Alternatively, this can be set with the following environment variable: " + agentInboundHostEnvKey
+
+	agentInboundHostEnvKey = "ARIESD_INBOUND_HOST"
 
 	agentInboundHostExternalFlagName = "inbound-host-external"
 
 	agentInboundHostExternalFlagShorthand = "e"
 
 	agentInboundHostExternalFlagUsage = "Inbound Host External Name:Port." +
 		" This is the URL for the inbound server as seen externally." +
-		" If not provided, then the internal inbound host will be used here."
+		" If not provided, then the internal inbound host will be used here." +
+		" Alternatively, this can be set with the following environment variable: " + agentInboundHostExternalEnvKey
+
+	agentInboundHostExternalEnvKey = "ARIESD_INBOUND_HOST_EXTERNAL"
+
+	agentDBPathFlagName = "db-path"
 
-	// AgentDBPathFlagName is the flag name for the database path command line argument.
-	AgentDBPathFlagName = "db-path"
+	agentDBPathFlagShorthand = "d"
 
-	// AgentDBPathFlagShorthand is the flag shorthand name for the database path command line argument.
-	AgentDBPathFlagShorthand = "d"
+	agentDBPathFlagUsage = "Path to database." +
+		" Alternatively, this can be set with the following environment variable: " + agentDBPathEnvKey
 
-	// AgentDBPathFlagUsage is the flag usage text for the database path command line argument.
-	AgentDBPathFlagUsage = "Path to database"
+	agentDBPathEnvKey = "ARIESD_DB_PATH"
 
-	// AgentWebhookFlagName is the flag name for the webhook command line argument.
-	AgentWebhookFlagName = "webhook-url"
+	agentWebhookFlagName = "webhook-url"
 
 	agentWebhookFlagShorthand = "w"
 
 	agentWebhookFlagUsage = "URL to send notifications to." +
-		" This flag can be repeated, allowing for multiple listeners."
+		" This flag can be repeated, allowing for multiple listeners." +
+		" Alternatively, this can be set with the following environment variable (in CSV format): " + agentWebhookEnvKey
+
+	agentWebhookEnvKey = "ARIESD_WEBHOOK_URL"
 
 	agentDefaultLabelFlagName = "agent-default-label"
 
 	agentDefaultLabelFlagShorthand = "l"
 
-	agentDefaultLabelFlagUsage = "Default Label for this agent. Defaults to blank if not set."
+	agentDefaultLabelFlagUsage = "Default Label for this agent. Defaults to blank if not set." +
+		" Alternatively, this can be set with the following environment variable: " + agentDefaultLabelEnvKey
+
+	agentDefaultLabelEnvKey = "ARIESD_DEFAULT_LABEL"
+
+	unableToStartAgentErrMsg = "unable to start agent"
```

Comment:
I realised a bunch of these were exported when they didn't need to be, so I fixed that. Removed those comments since they were really only there because the linter forced me to - they didn't really add anything.